### ISSUE TITLE
model.create(Unique constraint failed on the constraint: PRIMARY

### ISSUE THREAD

**Issue by hung101** [ookamiiixd/baileys-api#154]
I facing some messages does not store into database which get the error from the logs as following:
<img width="1703" alt="image" src="https://user-images.githubusercontent.com/6055412/220806861-a059f688-c406-4e2f-baa3-259897871ff6.png">

**Comment by ookamiiixd** [comment]
Yes, the store is not stable yet. I'll investigate more to see if there's any solution for this

**Comment by hung101** [comment]
> Yes, the store is not stable yet. I'll investigate more to see if there's any solution for this

thank you really appreciate. will wait for the update :)
